src/pages/[platform]/build-a-backend/add-aws-services/deletion-backup-resources/index.mdx

```diff
@@ -0,0 +1,136 @@
+
+Using underlying CDK construct properties you can modify resource configurations. This allows you to customize backend resources beyond what is offered via the `define*` functions.
+
+In the `amplify/backend.ts` create a `backend` object that exposes resource properties with objects for each of the components passed into the `defineBackend` function.
+
+```ts title="amplify/backend.ts"
+import { defineBackend } from '@aws-amplify/backend';
+import { auth } from './auth/resource';
+import { data } from './data/resource';
+
+const backend = defineBackend({
+  auth,
+  data
+});
+```
+
+## Enabling Deletion protection on a Auth resource
+
+For example, if you would like to enable Deletion protection on a Cognito user pool resource created by Amplify Auth.
+
+```ts title="amplify/backend.ts"
+backend.auth.resources.cfnResources.cfnUserPool.deletionProtection = "ACTIVE";
+```
+
+## Enabling Deletion protection on a Data resource
+
+For example, if you would like to enable Deletion protection on all DynamoDB tables created by GraphQL API.
+
+```ts title="amplify/backend.ts"
+const dataResources = backend.data.resources.cfnResources;
+for (const table of Object.values(dataResources.amplifyDynamoDbTables)) {
+  table.deletionProtectionEnabled = true;
+}
+```
+
+## Enabling Point-in-time recovery for DynamoDB tables
+
+For example, enabling Point-in-time recovery for all the DynamoDB tables created by GraphQL API. By default Point-in-Time recovery retains backups for 35 days.
+
+```ts title="amplify/backend.ts"
+const dataResources = backend.data.resources.cfnResources;
+for (const table of Object.values(dataResources.amplifyDynamoDbTables)) {
+  table.pointInTimeRecoveryEnabled = true;
+}
+```
+
+## Enabling Backups for DynamoDB tables
+
+For example, if your DynamoDB tables requires backups that extend the default 35 days point-in-time recovery, AWS Backup service can be utilized to centralize and automate backups for DynamoDB tables. 
+The example below outlines a backup plan configured to run daily at midnight, for all DynamoDB tables. 
+
+```ts title="amplify/backend.ts"
+import { defineBackend } from "@aws-amplify/backend";
+import {
+  BackupPlan,
+  BackupPlanRule,
+  BackupResource,
+  BackupVault,
+} from "aws-cdk-lib/aws-backup";
+import { Schedule } from "aws-cdk-lib/aws-events";
+import { Duration } from "aws-cdk-lib/core";
+import { auth } from "./auth/resource";
+import { data } from "./data/resource";
+
+const backend = defineBackend({
+  auth,
+  data,
+});
+
+const backupStack = backend.createStack("backup-stack");
+const ITables = Object.values(backend.data.resources.tables);
```

Comment:
can we choose a different name for this? the `I<something>` convention is typically reserved for interfaces